docs/finality-provider-phase2.md

```diff
@@ -0,0 +1,589 @@
+use `file` or `os` backend.
+
+
+ There are three options for the keyring backend:
+
+ `test`: Stores keys unencrypted on disk. It’s meant for testing purposes and
+ should never be used in production.  
+ `file`: Stores encrypted keys on disk,
+ which is a more secure option than test but less secure than using the OS
+ keyring.  
+ `os`: Uses the operating system's native keyring, providing the
+ highest level of security by relying on OS-managed encryption and access
+ controls.
+
+This command will create a new key pair and store it in your keyring. The output
+should look similar to the below.
+
+``` shell
+- address: bbn19gulf0a4yz87twpjl8cxnerc2wr2xqm9fsygn9
+  name: finality-provider pubkey:
+  '{"@type":"/cosmos.crypto.secp256k1.PubKey",
+  "key":"AhZAL00gKplLQKpLMiXPBqaKCoiessoewOaEATKd4Rcy"}'
+  type: local
+```
+
+>Note: Please verify the `chain-id` from the Babylon RPC
+node [https://rpc.testnet5.babylonlabs.io/status]
+(https://rpc.testnet5.babylonlabs.io/status)
+
+ >The configuration below requires to point to the path where this keyring is
+ stored `KeyDirectory`. This `Key` field stores the key name used for
+ interacting with the babylon chain and will be specified along with
+ the `KeyringBackend`field in the next step. So we can ignore the setting of the
+ two fields in this step.
+
+Once the node is initialized with the above command. It should generate a
+`fpd.config` Edit the `config.toml` to set the necessary parameters with the
+below
+
+```shell 
+[Application Options] EOTSManagerAddress = 127.0.0.1:12582 RpcListener
+= 127.0.0.1:12581
+
+[babylon] Key = <finality-provider-key-name-signer> // the key you used above
+ChainID = bbn-test-5 RPCAddr = http://127.0.0.1:26657 GRPCAddr =
+https://127.0.0.1:9090 KeyDirectory = ./fpKey 
+``` 
+
+### Step 3: Verify the Key Import
+After importing, verify that your EOTS key was successfully loaded:
+
+```shell 
+eotsd keys list
+```
+
+You should see your EOTS key listed with the correct details, confirming that 
+it has been imported correctly.
+
+>Note: Make sure you're using the same key name and EOTS public key that were 
+registered in Phase 1.
+
+
+## Starting the Finality provider Daemon
+
+The finality provider daemon (FPD) needs to be running before proceeding with 
+registration or voting participation.
+
+Start the daemon with:
+
+``` shell
+fpd start  --home <path> 
+```
+An example of the `--home` flag is `--home ./fpKeys`.
+
+The command flags:
+- `start`: Initiates the FPD daemon
+- `--home`: Specifies the directory for daemon data and configuration
+
+The daemon will start the RPC server for CLI communication then begin listening 
+for incoming requests and finally initialize finality provider services
+
+You should see logs indicating successful startup:
+
+```
+[INFO] Starting finality provider daemon...
+[INFO] RPC server listening on...
+```
+
+>Note: Keep this terminal window open as the daemon needs to run continuously.
```

Comment:
This won't work for production loads, we should tell people to use a systemd file or something to keep this running continuously.